core/src/com/mygdx/game/config/GameConfigs.java

```diff
@@ -16,6 +16,8 @@ public class GameConfigs {
   public static final int UNIT_MAX = 7;
   public static final int SUBFIELD_MIN = 7;
   public static final int SUBFIELD_MAX = 8;
+  public static final int TECHNOLOGY_MIN = 8;
+  public static final int TECHNOLOGY_MAX = 10;
```

Comment:
ale mnie dziwi, że takie małe przedziały sobie dajesz, zamiast mieć 0-1000, 1001-2000, itd.

bo tak to jak będą dodawane nowe configi to będzie po prostu niewygodnie zmieniać to co chwilę

Reply:
Potem to zmienię tak czy siak musi być np. od 1 do 5 jak mamy tylko 5 fieldów bo gdzieś tam losuję configa z tego zakresu